pulumi-kubernetes-operator.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: pulumi-kubernetes-operator
+  version: 1.11.4
+  epoch: 0
+  description: A Kubernetes Operator that automates the deployment of Pulumi Stacks
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - go
+      - busybox
+      - git
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/pulumi/pulumi-kubernetes-operator.git
+      tag: v${{package.version}}
+      destination: ${{package.name}}
+      expected-commit: edb18f2520a0bc6ac86bc0dd27ee542cd1c8ebce
+
+  - working-directory: ${{package.name}}
+    pipeline:
+      - runs: |
+          # Original Go build args found in ./scripts/build.sh
+          CGO_ENABLED=0 go build -o "${{targets.destdir}}/usr/bin/${{package.name}}" \
+            -ldflags "-X github.com/pulumi/pulumi-kubernetes-operator/version.Version=v${{package.version}} -w -extldflags \"-static\"" \
+            -tags netgo ./cmd/manager/main.go
```

Comment:
Can we just run `./scripts/build.sh`?

Reply:
unfortunately no. the script currently hardcodes GOARCH: https://github.com/pulumi/pulumi-kubernetes-operator/blob/325893699dff023cefb261db0c446d8cf7408d16/scripts/build.sh#L5

Reply:
Do you think they'd accept a patch upstream 🤔 

(not blocking, this is fine)